### PR TITLE
support for IN clause

### PR DESCRIPTION
**Pull request by linjiacheng**

### CODE REVIEW

**Comment by henkmollema** [comment]
Thanks for this. However I find the current implementation too complex and fragile. Can you think of a solution without using reflection to determine which `Contains` method is called and without checking for types like `System.Runtime.CompilerServices.ExtensionAttribute`?

**Comment by linjiacheng** [comment]
I made some changes, not sure if it works.

File: test/Dommel.Tests/SupportDynamicExpressions.cs [hunk 1]
```diff
@@ -72,6 +72,21 @@ public void OrExpression()
             Assert.Equal(2, dynamicParameters.Get<int>("p2"));
         }
 
+        [Fact]
+        public void InExpression()
```
Comment (henkmollema) [inline test/Dommel.Tests/SupportDynamicExpressions.cs]: Could you add tests for `decimal`'s and `Guid`'s as well?

**Comment by henkmollema** [comment]
Thanks a lot @linjiacheng!

**Pull request merged**